java-file-formats/word-library/mail-merge/mail-merge-for-group.md

```diff
@@ -23,7 +23,7 @@ The region where the Mail merge operations are to be performed must be marked by
   
 For example, consider that you have a template document as shown.
 
-![Mail merge for a group](../MailMerge_images/Group_mail_merge_template.png)
+![Java File Formats Word Library Mail merge for a group Template](../MailMerge_images/java-file-formats-word-library-mail-merge-template.png)
```

Comment:
Seems image folder name casing also redirect to small casing. So, please change the folder name casing to small case.
example: 
https://help.syncfusion.com/java-file-formats/word-library/mailmerge_images/group_mail_merge_output.png

Check and update the changes in all places.

Reply:
Resolved